src/main/java/org/web3j/abi/datatypes/NumericType.java

```diff
@@ -43,4 +47,9 @@ public int hashCode() {
         result = 31 * result + (value != null ? value.hashCode() : 0);
         return result;
     }
+
+    @Override
```

Comment:
We need more specific Numeric type toString implementations - there are fixed point types, as well as signed versus unsigned that need to be taken into account.